bootloader/bootloader.go

```diff
@@ -152,6 +152,10 @@ type ManagedAssetsBootloader interface {
 	ManagedAssets() []string
 	// UpdateBootConfig updates the boot config assets used by the bootloader.
 	UpdateBootConfig(*Options) error
+	// CommandLine returns the kernel command line composed of the built-in
+	// list and extra arguments passed in arguments. The command line may be
+	// different when using a bootloader in the recovery partition.
+	CommandLine(extra []string) (string, error)
```

Comment:
Should this also define `RecoveryCommandLine`?

Reply:
No, I don't think it's needed. Internally we'd know whether it's dealing with a recovery bootloader since the right flags were passed when instantiating one.